plugin-packs/postcss-preset-env/FEATURES.md

```diff
@@ -19,6 +19,7 @@ The `ID` listed is the key for PostCSS Preset Env configuration in your project.
 | `dir-pseudo-class` | `:dir` Directionality Pseudo-Class | [example](https://preset-env.cssdb.org/features/#dir-pseudo-class) | [docs](https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-dir-pseudo-class#readme) |
 | `display-two-values` | Two values syntax for `display` | [example](https://preset-env.cssdb.org/features/#display-two-values) | [docs](https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-normalize-display-values#readme) |
 | `double-position-gradients` | Double Position Gradients | [example](https://preset-env.cssdb.org/features/#double-position-gradients) | [docs](https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-double-position-gradients#readme) |
+| `float-clear-logical-values` | Logical Values in float and clear | [example](https://preset-env.cssdb.org/features/#float-clear-logical-values) | [docs](https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-logical#readme) |
```

Comment:
The rest is missing from CSSDB I think I jumpstarted on this too soon. 

Once we start to release and we make V8 go into `main` it should be safer to add the rest

Reply:
I think we can do a minor version in CSSDB where we add the plugins as polyfills?

Reply:
On second thought, let's leave this as if for now.
`FEATURES.md` can be updated without requiring a release.

Doing `v8` first and then adding the links makes more sense :)